src/main/java/com/ably/kafka/connect/config/ChannelSinkConnectorConfig.java

```diff
@@ -177,6 +176,10 @@ public class ChannelSinkConnectorConfig extends AbstractConfig {
     private static final String CLIENT_CHANNEL_PARAMS_DOC = "Additional channel parameters used to configure the " +
         "behaviour of the channel. This should be specified in the form \"key1=value1,key2=value2,...\".";
 
+    public static final String SKIP_ON_KEY_ABSENCE = "skipOnKeyAbsence";
+    private static final String SKIP_ON_KEY_ABSENCE_DOC = "If true, it skips the record if the key has been provided as" +
```

Comment:
Feels like unnecessary string split and concatenation.

Reply:
As above

Reply:
https://github.com/ably/kafka-connect-ably/pull/89#discussion_r1010313991